src/views/Picklists.vue

```diff
@@ -43,11 +53,14 @@ export default defineComponent({
     IonPage,
     IonTitle,
     IonToolbar,
-    PicklistItem
+    PicklistItem,
+    IonItem
```

Comment:
Arrange components alphabetically.

Reply:
Arranged components alphabetically.
Thank you sir for reviewing my code.